app/scripts/controllers/transactions/pending-tx-tracker.js

```diff
@@ -79,7 +79,7 @@ export default class PendingTransactionTracker extends EventEmitter {
       try {
         await this._resubmitTx(txMeta, blockNumber)
       } catch (err) {
-        const errorMessage = err.message.toLowerCase()
+        const errorMessage = err.value.message.toLowerCase()
```

Comment:
Could we keep using `err.message` as a fallback? The `message` property is the standard place or error messages to go. I'm guessing `value.message` is specific to these RPC errors.

```suggestion
        const errorMessage = err.value?.message?.toLowerCase() || err.message?.toLowerCase()
```